src/Laravel/Eloquent/Filter/DateFilter.php

```diff
@@ -44,4 +82,26 @@ public function getSchema(Parameter $parameter): array
     {
         return ['type' => 'date'];
     }
+
+    public function getOpenApiParameters(Parameter $parameter): OpenApiParameter|array|null
+    {
+        $in = $parameter instanceof QueryParameter ? 'query' : 'header';
+
+        return [
+            new OpenApiParameter(name: $parameter->getKey().'[eq]', in: $in),
+            new OpenApiParameter(name: $parameter->getKey().'[gt]', in: $in),
+            new OpenApiParameter(name: $parameter->getKey().'[lt]', in: $in),
+            new OpenApiParameter(name: $parameter->getKey().'[gte]', in: $in),
+            new OpenApiParameter(name: $parameter->getKey().'[lte]', in: $in),
+        ];
+    }
+
+    public function getDateTime($value): ?\DateTimeImmutable
```

Comment:
Can't this method be private? Also, if possible, please add a type hint for the parameter.